File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/codex-sh/sieve.java
--- 
+++ 
@@ -1,7 +1,14 @@
 public static ArrayList<Integer> sieve(Integer max) {
     ArrayList<Integer> primes = new ArrayList<Integer>();
     for (int n=2; n<max+1; n++) {
-        if (any(list_comp(n, primes))) {
+        boolean prime = true;
+        for (int p : primes) {
+            if (n % p == 0) {
+                prime = false;
+                break;
+            }
+        }
+        if (prime) {
             primes.add(n);
         }
     }

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/codex-sl/sieve.py
--- 
+++ 
@@ -1,6 +1,6 @@
 def sieve(max):
     primes = []
     for n in range(2, max + 1):
-        if any(n % p > 0 for p in primes):
+        if all(n % p for p in primes):
             primes.append(n)
     return primes
